tests/test_logging/basic_scenes.py

```diff
@@ -5,15 +5,7 @@
 
 class SquareToCircle(Scene):
```

Comment:
Nitpick: `SquareAndCircle` maybe?

Reply:
I don't think so, as this is the same as `SquareToCircle` scene ;)